src/index.html

```diff
@@ -5,9 +5,45 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <h1 hidden>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__photo"
+        src="./images/imac.jpeg"
+        alt="Apple iMac27 Monoblock"
+      >
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <p class="card__code">Product code: 195434</p>
+      <div class="card__review">
+        <div class="card__review-stars stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__review-text">Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-label">Price:</p>
+        <p class="card__price-amount">$2,199</p>
+      </div>
+      <a
+        href="#"
+        data-qa="hover"
+        class="card__link"
+      >Buy</a>
```

Comment:
```suggestion
      >
        Buy
      </a>
```
Consider fixing indentations according to the code style guide